src/DynamicRupture/FrictionLaws/GpuImpl/AgingLaw.h

```diff
@@ -0,0 +1,44 @@
+#ifndef SEISSOL_GPU_AGINGLAW_H
+#define SEISSOL_GPU_AGINGLAW_H
+
+#include "DynamicRupture/FrictionLaws/GpuImpl/SlowVelocityWeakeningLaw.h"
+
+namespace seissol::dr::friction_law::gpu {
+
+/**
+ * This class was not tested and compared to the Fortran FL3. Since FL3 initialization did not work
+ * properly on the Master Branch. This class is also less optimized. It was left in here to have a
+ * reference of how it could be implemented.
+ */
```

Comment:
Remove this comment. FL3 has been reintroduced in the Fortran parts and we have a proper reference now.

Reply:
done.